### DISCUSSION TITLE
Enable specifying Intune app name

### DISCUSSION THREAD

**Discussion by fergusstrachan** [svrooij/WinTuner#43] [Ideas]
Command: `Deploy-WtWin32App`

Hi Stephan,

Just trying this out. Seems great so far, really slick.

What would be really useful is to be able to specify the name of the resulting Win32 app, rather than taking the Name from Winget directly. I like to separate out these apps with a prefix while testing.

It seems like the module requires PS7+ as well, which I didn't see in the docs. Not an expert so might have missed something obvious. :-)

Appreciating your hard work.
F.

**Comment by svrooij** [comment]
## PowerShell Version

You're totally right about the required PowerShell Version, this was not clear before. The idea was to make a PowerShell module that was compatible with both, but this is way more challenging then I thought. The [next](https://github.com/svrooij/WingetIntune/pull/45) version will restrict this and I'll update the docs.

## Specify name on deploy

Yes, why not. `-OverrideAppName <string>` coming up in the next version as well. 

## Check out version 0.8.3

[Latest version](https://www.powershellgallery.com/packages/WinTuner/0.8.3) this has support for overriding the name and the minimum version is update to 7.4

**Comment by fergusstrachan** [comment]
Brilliant stuff, thanks Stephan, that's even better now!

Another suggestion though... ;-)
Duplicate avoidance - it'll create duplicate Win32App entries for the same app and version. Might be worth avoiding this by default, perhaps with an override option?

Ferg.
